src/content/blog/posts/github-one-stop-shop.md

```diff
@@ -0,0 +1,215 @@
+
+### Presentations
+
+[Marp](https://marp.app/) is an open-source ecosystem for creating slide decks in Markdown, which can be exported to various formats, including PDF and HTML. Similar to Excalidraw, GitHub does not provide native support, but you can save the Markdown files in a `/docs/presentation` folder. The [marp-to-pages](https://github.com/ralexander-phi/marp-to-pages) GitHub template repository is great for starting out and includes a GitHub action for exporting to HTML and publishing to GitHub Pages. The [VS Code extension](https://marketplace.visualstudio.com/items?itemName=pomdtr.excalidraw-editor) makes it easier to edit and preview Markdown files locally.
+
+While there's a bit of a learning curve, once you've decided on a format, it's very intuitive. We've successfully used Marp to create slide decks when [evaluating Iceberg](https://github.com/moj-analytical-services/iceberg-evaluation/tree/main/docs/presentations). Additionally, I've exported the `README.md` to HTML and hosted it on [GitHub Pages](https://moj-analytical-services.github.io/iceberg-evaluation/) for better visibility. Note that you shouldn't commit [generated files](https://homes.cs.washington.edu/~mernst/advice/version-control.html), but it's fine for exploratory projects. For production code, you should use something like the [marp-to-pages](https://github.com/ralexander-phi/marp-to-pages/blob/main/.github/workflows/marp-to-pages.yml) GitHub action, which generates PDF and HTML files as part of a pull request or merge.
+
+# Conclusion
+
+Historically, GitHub has been mainly valued for source code management. Recent enhancements to the GitHub ecosystem, coupled with various open-source tools, make it possible to integrate code, automation, tracking, and documentation within one platform. While there may be initial challenges, the long-term benefits include increased efficiency, consistency, and transparency.
```

Comment:
As we discussed earlier, could you add a small extra paragraph along the lines of... We've found this set-up beneficial when we've tried it but it may not work in all circumstances.

Reply:
done